pkg/kubelet/cadvisor/helpers_linux.go

```diff
@@ -43,6 +46,17 @@ func (i *imageFsInfoProvider) ImageFsInfoLabel() (string, error) {
 	return "", fmt.Errorf("no imagefs label for configured runtime")
 }
 
+func (i *imageFsInfoProvider) ContainerFsInfoLabel() (string, error) {
+	// This is a temporary workaround to get stats for cri-o from cadvisor
+	// and should be removed.
+	// Related to https://github.com/kubernetes/kubernetes/issues/51798
+	if i.runtimeEndpoint == CrioSocket || i.runtimeEndpoint == "unix://"+CrioSocket {
+		return LabelCrioContainers, nil
+	}
```

Comment:
can we define a helper function for this so we can change it in one spot later?

Reply:
I added a helper function for this.